modules/javafx.graphics/src/main/java/com/sun/javafx/application/PlatformImpl.java

```diff
@@ -358,19 +353,6 @@ private static void addExportsToFXCanvas(Class<?> fxCanvasClass) {
 
     // FXCanvas-specific initialization
     private static void initFXCanvas() {
-        // Verify that we have the appropriate permission
```

Comment:
this might create a (trivial) merge conflict

Reply:
It should be easy to resolve. Thanks for pointing it out.